biodatasets/swedish_medical_ner/swedish_medical_ner.py

```diff
@@ -93,106 +93,6 @@
-                # In the original data, entities are wrapped with brackets/braces/parentheses.
-                # Here we identify entities with brackets:
-                matches = re.findall(_ents, sentence, overlapped=True)
-                # Remove brackets/braces/parentheses
-                words = [re.sub(r"[{}\[\]\(\)]", "", m) for m in matches]
-                # Match entity types
-                types = [find_type(match[0], match[-1]) for match in matches]
-                # Add space to the entity and the word on its left (if there isn't a space in between), e.g.
-                # "jukdom(KOL)(lungfunktionsundersökning)" -> "jukdom (KOL) (lungfunktionsundersökning)"
-                clean_sent = re.sub(r"(\w|\)|\]|})([\(\[{])", "\\1 \\2", sentence)
-                # Add space to the entity and the word on its right (if there isn't a space in between), e.g.
-                # "(kronisk bronkit)och" -> "(kronisk bronkit) och"
-                clean_sent = re.sub(r"([\)\]}])(\w|\(|\[|{)", "\\1 \\2", clean_sent)
-                # Remove the brackets/braces/parentheses around each entity
-                clean_sent = re.sub(r"[{}\[\]\(\)]", "", clean_sent)
-
-                # Remove brackets/braces/parentheses from sentences and find the char offsets of each entity
-                # in the cleaned sentence
-                try:
-                    targets = [
-                        {
-                            "start": m.start(2),
-                            "end": m.end(2),
-                            "text": clean_sent[m.start(2) : m.end(2)],
-                        }
-                        for m in map(lambda word: re.search(f"(^|[^\w]+)({word})($|[^\w]+)", clean_sent), words)
-                    ]
-                # Skip wrongly formatted entities
-                # e.g. "rån patienter med (Barretts ){e)s)o)f)a)g)u)s),} påverkas av korta pulse"
-                # e.g. "let samt (imperforerad ){a)n)u)s).} Möten mellan himmel och jord"
-                # I have decided to ignore these cases since they are difficult to fix and would require the use
-                # of regex thus increase processing time.
-                except:
-                    continue
-
-                if targets:
-                    documents.append(
-                        {
-                            "sid": "s" + str(s_id),
-                            "sentence": clean_sent,
-                            "entities": [
-                                {
-                                    "start": targets[i]["start"],
-                                    "end": targets[i]["end"],
-                                    "text": targets[i]["text"],
-                                    "type": types[i],
-                                }
-                                for i in range(len(targets))
-                            ],
-                            "normalized": {"db_name": db_name, "db_id": "d" + str(db_id)},
```

Comment:
@bwang482 
I don't think these entities don't actually have normalizations (defined as a mapping back to a canonical identifier in a knowledge base). I would leave normalizations empty.